<launch>
    <arg name="veh" doc="Name of vehicle. ex: megaman"/>
    <arg name="param_file_name" default="default" doc="Specify a param file. ex:megaman." />
    <arg name="pkg_name" value="lf_pure_pursuit"/>
    <arg name="node_name" default="lf_pure_pursuit_node"/>
    <arg name="verbose" default="false" />

    <group ns="$(arg veh)">
        <node name="$(arg node_name)" pkg="$(arg pkg_name)" type="$(arg node_name).py" output="screen" required="true">
            <rosparam command="load" file="$(find lf_pure_pursuit)/config/$(arg node_name)/$(arg param_file_name).yaml"/>
            <param name="verbose" value="$(arg verbose)"/>
        </node>
    </group>
</launch>
